### DISCUSSION TITLE
Airdrop token dağıtımı

### DISCUSSION THREAD

**Discussion by mmuatt86** [IBAX-io/go-ibax#1154] [Soru Cevap - Türkçe]
İbax Network Airdrop dağılımı çok adaletsiz. Sizce nasıl?

**Comment by ibaxio** [comment]
Niye ya?

**Comment by ozkankalkan** [comment]
referans puanı ile üst sıralara geçmek isteyenler fake hesaplar açarak puan yükseltti. bence de bu konuda adaletsizlik olacak. zaten yazılım ve kodlama bilenler burda bizden üstte olacaktı, bir de fake hesap açıp referans kasanlar önümüze geçti. burda bir adaletsizlik oldu ama maalesef böyle airdrop etkinliklerinde olağan kabul edilen şeyler.  yine de konuyu açan arkadaşa katılıyorum.

**Comment by mmuatt86** [comment]
Sebebi şu tam olarak.
İkramiye A için 👍 Toplam 1 500 0000 ibxc 
📌İlk 1000 kişi =500 er ibxc =500 000  burada sıkıntı yok.
📌1000 den sonraki kişiler için 300k. Twitter da 69 k var 4.4 filan düşer adam başı. Çook düşük. 
📌 45 000 puanı geçenler arasından sadece ve sadece 7 kişiden herbiri, hazır olun, tamı tamına 100 er binden 700 000 ibxc alacak. Diğerlerine oranla aşırı fazla ve bu puana githup ta kodlama işinden anlayanlar ve twitterda şanslı kişi olanlar ulaşabilecek sanırım. 

İkramiye B için 3 500 0000 ibxc 
📌150 000 ibxc referans ödülü : sistem güzel sıkıntı yok. 
📌 350 000 ibxc referans sıralamasında ki ilk ona verilecek ve sadece 300 bini 1. Olan kullanıcıya verilecek. 2.ise 25 k alacak ve azalarak devam edecek. Farklar aşırı fazla 1. Ve 2. arasında ne kadar fark olabilir derseniz işte bu kadar olabilir. 
📌Hani geriye tam tamına 3 000 000 ibxc varya aslında o yok :) Çünkü o havuzdan geriye bu tarih itibariyle sadece 150 000 ibxc kaldı. Çünkü airdrop başladığı andan itibaren bitimine kadar geçen 60 günde her gün bir şanslı kullanıcıya 50 000 adet verildi ve sona gelindi 3 kişiye daha verilip bitecek. Yani 60 kişi de 3 milyon ibxc paylaştı. 

Özetle :  1 kişi 300 bin, 7 kişi 100 er bin, 60 kişi 50 şer bin,  1 kişi 25 bin vs. Toplamda 70-75 kişi kayda değer bir airdrop ödülü alacak. Geriye kalan avucunu yalayacak ve diline 3-5 tane airdrop yapışırsa şükredecek. 
Dağıtım biraz daha adaletli olabilirdi.
